datafusion/core/tests/sql/expr.rs

```diff
@@ -200,6 +200,14 @@ async fn binary_bitwise_shift() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_comparison_func_expressions() -> Result<()> {
```

Comment:
cool, those funcs were in my list, thanks for taking this. 
please add more test scenariuos? with nulls, different datatypes